models/actions.go

```diff
@@ -62,6 +61,37 @@ func (a *Action) GetValue() interface{} {
 	return nil
 }
 
+func (a *Action) GetProto() proto.Message {
```

Comment:
Should this be added to the ActionInterface interface?

Reply:
Added in [e561432](https://github.com/cloudfoundry/bbs/pull/109/commits/e561432dc8eee13c6ffca3f5956e97ebf7d3f7bc)